onboarding/models.py

```diff
@@ -279,6 +279,9 @@ def hubspot_data(self):
 
     def __unicode__(self):
         return self.label
```

Comment:
Remove this, it is python 2 specific, isn't it?

Reply:
I also deleted/replaced in the remaining tables of the `Onboarding` application